flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -223,4 +226,10 @@ private Object restorePartitionValueFromFromType(String valStr, DataType type) {
 		throw new FlinkHiveException(
 				new IllegalArgumentException(String.format("Can not convert %s to type %s for partition value", valStr, type)));
 	}
+
+	@Override
+	public String explainSource() {
+		return super.explainSource() + String.format(" PartitionPruned: %s, partitionNums: %d",
```

Comment:
Can we also add the db/table name here? It'll be helpful to debugging.

Reply:
Oh, just find the comment. +1 to add db/table name. 
But I'm fine if we add it later. It doesn't break the correctness.

Reply:
Sure I'm OK with that too.